src/main/java/edu/cornell/kfs/concur/service/ConcurAccessTokenService.java

```diff
@@ -0,0 +1,16 @@
+package edu.cornell.kfs.concur.service;
+
+
+public interface ConcurAccessTokenService {
+    
+    String getConcurAccessToken();
```

Comment:
Since the Interface has Concur in it, I wonder if Concur is also necessary in these method names.